index.rst

```diff
@@ -1,59 +1,367 @@
+The default role is the user's unix username, which provides write access only to ``u/<user>`` collections.
+These guards against careless fingers, not careless brains - we will not attempt to restrict which roles a user can assume *at the butler client level*.
+
+.. note::
+
+   This proposal intentionally makes no mention of the RFC process that is currently in place for ``/datasets`` for Gen2, or the ``/project`` filesystem.
+
+   In practice, RFCs for most modifications - certainly routine ingests or calibration updates - almost never exceed our `Empowerment of DM team members`_ criteria, and asking for sysadmins who are not domain experts to do the actual work both increases friction and increases the chance something will go wrong (or, rather, the chances that if something does go wrong, it is not fixed immediately).
+
+   This proposal makes no objection to having a separate ``/project`` filesystem for files.
+   But using ``/project`` and symlinks as a way to work around restrictions on write access to appropriate subsets of ``/datasets`` is just that - a workaround - and one that makes it more difficult than it ought to be to find things.
+   If filesystem-level controls (or quotas, etc.) really are necessary even for ``/datasets`` (note that we already allow for filesystem-level controls by symlinking raws from ``/lsstdata`` and ``/external-raw``), I think this is something we can tolerate, but regular directory permission controls within ``/datasets`` would be preferable.
+   If we do use symlinks from ``/project``, we should at least ensure that a::
+
+       /datasets/repo/u/<user> -> /project/<user>/.datarepo
+
+   link (or equivalent) is automatically created with the right permissions for all users, and encourage access only via that symlink (hence the hidden directory as a target).
+
+.. _Empowerment of DM team members: https://developer.lsst.io/team/empowerment.html#empowerment-of-dm-team-members>
+
+
+Personal and test-package repositories
+======================================
+
+This proposal is primarily concerned with long-lived, shared data repositories of the sort that will exist not just at NCSA, but at the IDF, SLAC, CCIN2P3, and other major LSST data facilities.
+
+Small repositories (typically backed by SQLite) are also expected to be common, especially for small-scale CI and local development.
+These repositories should follow the same naming patterns whenever possible, but will generally not need as many levels of indirection to guard against future changes or collections, and many of the collections defined here as ``CHAINED`` or ``TAGGED`` collections can instead be safely defined directly as ``RUN`` collections instead.
```

Comment:
This guidance is a bit vague, especially since we're not used to the repository types yet. It's not obvious to new users when you can/cannot use a `RUN` collection for technical reasons (cf. my confusion on [`#dm-middleware`](https://lsstc.slack.com/archives/C2JPT1KB7/p1601312022091100) when `CALIBRATION` collections were introduced).

Do you simply mean that we can drop all the "<ticket>" branches from the scheme, and that any `CHAINED` collections that depended on them become `RUN`/`CALIBRATION`? Might it be worth making a copy of Table 1 for this section?

Reply:
> Do you simply mean that we can drop all the "<ticket>" collections from the scheme, and that any CHAINED collections that depended on them become RUN/CALIBRATION? Might it be worth making a copy of Table 1 for this section?

You, that's essentially what I meant.  I don't want to be too prescriptive on this point, but another table as an example is a good idea (but I may defer doing that until RFC adoption and the long-term home for this content is identified).